react/features/whiteboard/index.tsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable import/order */
+/* eslint-disable lines-around-comment */
+
+/* eslint-ignore */
+// @ts-ignore
+import { ExcalidrawApp } from '@jitsi/excalidraw';
+
+import React, { useCallback, useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import clsx from 'clsx';
+
+// @ts-ignore
+import Filmstrip from '../../../modules/UI/videolayout/Filmstrip';
+// @ts-ignore
+import { getVerticalViewMaxWidth } from '../filmstrip/functions.web';
+// @ts-ignore
+import { getToolboxHeight } from '../toolbox/functions.web';
+// @ts-ignore
+import { getCollabLink, getCollabServerUrl, getUsernameStatus, isWhiteboardVisible } from './functions';
+// @ts-ignore
+import { shouldDisplayTileView } from '../video-layout/functions.any';
+
+import { setUsernameStatus } from './actions';
+// @ts-ignore
+import { getLocalParticipant } from '../base/participants';
+import { USERNAME_UPDATE_STATUS } from './constants';
+
+const HEIGHT_OFFSET = 80;
+
+declare const interfaceConfig: any;
+
+interface IDimensions {
+
+    /* The height of the component. */
+    height: string;
+
+    /* The width of the component. */
+    width: string;
+}
+
+/**
+ * The Whiteboard component.
+ *
+ * @returns {JSX.Element} - The React component.
+ */
+const Whiteboard: () => JSX.Element = () => {
```

Comment:
This component should not be in index.tsx, drop this file and put it in components/Whiteboard.tsx

Reply:
Yup, I was planning to, thanks!